### PR TITLE
Add default enabled admission control plugins

### PR DESCRIPTION
**Pull request by charrywanganthony**
According to [kubernetes/issue #66979](https://github.com/kubernetes/kubernetes/issues/66979)

ref: [kubernetes/pr #65739](https://github.com/kubernetes/kubernetes/pull/65739)
ref: [kubernetes/pr #58684](https://github.com/kubernetes/kubernetes/pull/58684)

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -81,6 +81,14 @@ The Kubernetes API server flag `disable-admission-plugins` takes a comma-delimit
 kube-apiserver --disable-admission-plugins=PodNodeSelector,AlwaysDeny ...
 ```
 
+## What plugins are enabled by default?
+
+The default enabled admission plugins are: 
+```
+NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota
+```
+The `Priority` admission plugin is enabled by default when using `--enable-admission-plugins`. If using `--admission-control` to fully specify the set of admission plugins, the `Priority` admission plugin should be added if using the `PodPriority` feature, which is enabled by default in 1.11.
```
Comment (CaoShuFeng) [inline content/en/docs/reference/access-authn-authz/admission-controllers.md]: --admission-control is deprecated.
Not sure we want to describe a deprecated flag in document.

Comment (charrywanganthony) [reply content/en/docs/reference/access-authn-authz/admission-controllers.md]: @CaoShuFeng  Simplified.

**Comment by zparnold** [comment]
@charrywanganthony Is everything here listed the default for 1.11? Are we creating a way to maintain these through their lifecycle?

**Comment by charrywanganthony** [comment]
[Kubernetes/pr #67011](https://github.com/kubernetes/kubernetes/pull/67011/files) has been merged.  @zparnold WDYT about this pr now?

File: content/en/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -81,6 +81,20 @@ The Kubernetes API server flag `disable-admission-plugins` takes a comma-delimit
 kube-apiserver --disable-admission-plugins=PodNodeSelector,AlwaysDeny ...
 ```
 
+## What plugins are enabled by default?
```
Comment (zacharysarah) [inline content/en/docs/reference/access-authn-authz/admission-controllers.md]: s/What/Which

File: content/en/docs/reference/access-authn-authz/admission-controllers.md [hunk 3]
```diff
@@ -81,6 +81,20 @@ The Kubernetes API server flag `disable-admission-plugins` takes a comma-delimit
...
+
+You can get the enabled admission plugins in result of:
```
Comment (zacharysarah) [inline content/en/docs/reference/access-authn-authz/admission-controllers.md]: Replace this line with:
```
To see which admission plugins are enabled:
```

**Pull request merged**